src/app/scopes.ts

```diff
@@ -246,6 +260,20 @@ export const PermissionScopes: PermissionScope[] = [
     preview: false,
     admin: false,
   },
+  {
+    name: "ProgramControl.Read.All",
+    description: "Read all programs that user can access",
```

Comment:
Add the missing indefinite article "Read all programs that a user can access"

Reply:
Fixed.